### DISCUSSION TITLE
XMLSec 1.3.6 (rc1)

### DISCUSSION THREAD

**Discussion by lsh123** [lsh123/xmlsec#843] [Announcements]
The **XML Security Library 1.3.6** release includes the following changes:

- (xmlsec-openssl) Fixed build if OpenSSL 3.0 doesn't have engines support enabled.
- (xmlsec-mscng, xmlsec-mscrypto) Added support for multiple trusted certs with the same subject.
- (windows) Disabled iconv support by default (use 'iconv=yes' option for 'configure.js' to re-enable it).
- Several other small fixes (see [more details](https://github.com/lsh123/xmlsec/commits/master)).

Please test the release [candidate](https://www.aleksey.com/xmlsec/download/xmlsec1-1.3.6-rc1.tar.gz) ([signature](https://www.aleksey.com/xmlsec/download/xmlsec1-1.3.6-rc1.sig)) and let me know if you see any issues!

<hr /><em>This discussion was created from the release <a href='https://github.com/lsh123/xmlsec/releases/tag/1.3.6-rc1'>XMLSec 1.3.6 (rc1)</a>.</em>

**Comment by vmiklos** [comment]
odfsig integration tests found no issues.

**Reply by lsh123** [reply]
Thank you!
